modules/gatsby-theme-ocular/src/gatsby-config/get-gatsby-config.js

```diff
@@ -74,46 +110,16 @@ module.exports = function getGatsbyConfig(config) {
       'gatsby-plugin-sharp',
       'gatsby-transformer-sharp',
 
-      // Transforms markdown (.md) nodes, converting the raw markdown to HTML
+      // Register plugins used by MDX
+      ...remarkPlugins,
```

Comment:
- These were nested plugins for transformer-remark, are they now OK to have on top level? Do they actually do anything there?
- Or is MDX plugin completely taking over from the remark transformer?
- Or is the comment correct, they need to be listed twice?? 

Worth being a bit more verbose in the comment?

Reply:
`gatsby-plugin-mdx` replaces `gatsby-transformer-remark`:

https://www.gatsbyjs.org/docs/mdx/migrate-remark-to-mdx/#replacing-gatsby-transformer-remark-with-gatsby-plugin-mdx

The plugin requires all sub-plugins be registered twice:

https://www.gatsbyjs.org/docs/mdx/plugins/